rail_scripts/rail-preprocess-parquet

```diff
@@ -83,7 +83,8 @@ def parse_cmdline():
                          rows=rows, flux=args['--flux'],
                          output_template=args['--output-template'],
                          ra=args['--ra'], 
-                         dec=args['--dec'])
+                         dec=args['--dec'],
+                         objectId=args['--objectId'])
```

Comment:
The new `args` key is missing in the docopt string. Executing this line results in the following error:

```
Traceback (most recent call last):
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 237, in <module>
    if __name__ == '__main__': main()
                               ^^^^^^
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 234, in main
    cfg = parse_cmdline()
          ^^^^^^^^^^^^^^^
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 87, in parse_cmdline
    objectId=args['--objectId'])
             ~~~~^^^^^^^^^^^^^^
KeyError: '--objectId'
```

To make it work, the parameter must be added to the script's docstring, for example:
```
    --objectId=<objectId>             Set the name of the object ID column.
```